packages/api-request/src/middlewares/http-v1.js

```diff
@@ -0,0 +1,19 @@
+function httpV1Middleware( options, next ) {
+	const newOptions = { ...options };
```

Comment:
Why do we create a shallow clone _before_ the `if` condition where it becomes relevant? Wasted effort for the majority of cases?

Reply:
https://github.com/WordPress/gutenberg/pull/11332